src/Debugger.cs

```diff
@@ -393,6 +373,17 @@ static void EnsureCreated()
                     Log.Notice("Inferior thread '{0}' ('{1}') exited",
                                e.Thread.Id, e.Thread.Name);
                 };
+
+                _session.TargetEvent += (sender, e) =>
```

Comment:
By the way, there is already a handler attached for `TargetEvent` earlier in the file. Could just move this code there.